module/admin/helper.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * name : v1.js
+ * author : Prajwal
+ * created-date : 15-Apr-2024
+ * Description : Admin.
+ */
+
+module.exports = class Admin{
+    static createIndex(collection, keys) {
```

Comment:
@Prajwal17Tunerlabs Please add function signature

Reply:
i have added function signature to the said function